transcendence-app/src/app.service.ts

```diff
@@ -3,6 +3,6 @@ import { Injectable } from '@nestjs/common';
 @Injectable()
 export class AppService {
   getHello(): string {
-    return `Hello ${process.env.TRANSCENDENCE_APP_NAME}!`;
+    return `Hello World!`;
```

Comment:
Reverted this change because it will be removed soon and was the easiest way to fix broken test without having to pass any ENVs (not needed for now)